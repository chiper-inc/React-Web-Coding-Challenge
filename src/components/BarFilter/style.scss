.bar-filter {
    @apply  mt-8 flex items-center;

    .content-search{
        @apply mr-4;
        .input-search{
            @apply border-2 border-gray-400 rounded w-full p-1;
        }
    }
}