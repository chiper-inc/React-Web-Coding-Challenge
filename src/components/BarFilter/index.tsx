import { useState } from 'react';
import './style.scss';

const BarFilter = () => {
  return (
    <div className="bar-filter">
      <div className="content-search w-full md:w-3/12">
        <input
          className="input-search"
          placeholder="Search case descriptions"
        />
      </div>
      <div className="content-range-date"></div>
    </div>
  );
};

export default BarFilter;
