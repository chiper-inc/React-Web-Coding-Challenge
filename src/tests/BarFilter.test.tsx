import React from 'react';
import { shallow } from 'enzyme';
import BarFilter from '../components/BarFilter';

test('renders the component BarFilter Test', () => {
  const component = shallow(
    <BarFilter titleSearch="" setTitleSearch={() => console.log('')} />
  );
  expect(component).toMatchSnapshot();
});
