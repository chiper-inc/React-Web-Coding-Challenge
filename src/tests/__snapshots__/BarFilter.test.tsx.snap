// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the component BarFilter 1`] = `
<div
  className="bar-filter"
>
  <div
    className="content-search w-full md:w-3/12"
  >
    <input
      className="input-search"
      onChange={[Function]}
      placeholder="Search case descriptions"
      value=""
    />
  </div>
  <div
    className="content-range-date"
  />
</div>
`;

exports[`renders the component BarFilter Test 1`] = `
<div
  className="bar-filter"
>
  <div
    className="content-search w-full md:w-3/12"
  >
    <input
      className="input-search"
      onChange={[Function]}
      placeholder="Search case descriptions"
      value=""
    />
  </div>
  <div
    className="content-range-date"
  />
</div>
`;
